lib/3_add_up_numbers.rb

```diff
@@ -5,5 +5,5 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def add_up_numbers(arr)
-  # Write your code here.
+  arr.inject(0, :+)
```

Comment:
Nice! I notice this works whether the parameters passed to `#inject` are `(0, :+)` or just `(:+)`. Why do you think that is? (I think reading some blog posts may be more helpful for thinking about this than documentation).